presto-main/src/main/java/com/facebook/presto/operator/aggregation/CountColumn.java

```diff
@@ -107,7 +107,7 @@ private static BuiltInAggregationFunctionImplementation generateAggregation(Type
                 metadata,
                 classLoader);
         return new BuiltInAggregationFunctionImplementation(NAME, inputTypes, ImmutableList.of(intermediateType), BIGINT,
-                true, false, metadata, accumulatorClass, groupedAccumulatorClass);
+                true, false, true, metadata, accumulatorClass, groupedAccumulatorClass);
```

Comment:
you need to override the method isCalledOnNullInput() from SqlFunction/BuiltInFunction instead of setting this to true in the implementation (here and throughout)